packages/renderer/src/lib/container/container-utils.ts

```diff
@@ -32,17 +32,29 @@ import { ContainerGroupInfoTypeUI } from './ContainerInfoUI';
 
 export class ContainerUtils {
   getName(containerInfo: ContainerInfo) {
-    // part of a compose ?
-    const composeService = containerInfo.Labels?.['com.docker.compose.service'];
-    if (composeService) {
-      const composeContainerNumber = containerInfo.Labels?.['com.docker.compose.container-number'];
-      if (composeContainerNumber) {
-        return `${composeService}-${composeContainerNumber}`;
-      }
-    }
+    // If the container has no name, return an empty string.
     if (containerInfo.Names.length === 0) {
       return '';
     }
+
+    // Safely determine if this is a compose project or not by checking the "service" or project label.
+    if (containerInfo.Labels?.['com.docker.compose.service'] || containerInfo.Labels?.['com.docker.compose.project']) {
```

Comment:
Looks to me that checking `containerInfo.Labels?.['com.docker.compose.service']` is not necessary. 
If `containerInfo.Labels?.['com.docker.compose.service']` is defined and `containerInfo.Labels?.['com.docker.compose.project']` is undefined we get inside the if, but then the second if is skipped.

Reply:
Yeah you are right. We could just do `containerInfo.Labels?.['com.docker.compose.project']`